.github/workflows/ci.yaml

```diff
@@ -43,10 +43,11 @@ jobs:
           poetry config virtualenvs.path ~/poetryenvs
           poetry install
 
-      - name: Run pylint
+      - name: Run Ruff
         shell: bash
         run: |
-          poetry run pylint hume --rcfile pyproject.toml
+          ruff check .
```

Comment:
This will probably need to be `poetry run ruff check .`

Reply:
You're right, fixed: https://github.com/HumeAI/hume-python-sdk/pull/150/commits/d42b22ec178912bc9c40d1cf37dd55df38ddf18e